File: TCDNew-master/src/ui.wince/Cen.Wms.Client/Models/Dtos/PurchaseTaskLineUpdatePostReq.cs
namespace Cen.Wms.Client.Models.Dtos
{
    public class PurchaseTaskLineUpdatePostReq
    {
        public string PurchaseTaskId { get; set; }
        public string ProductBarcode { get; set; }
        public string CurrentPalletCode { get; set; }
        public PurchaseTaskLineUpdateDto PurchaseTaskLineUpdate { get; set; }
    }
}

File: TCDNew-master/src/ui.android/lib/models/dtos/PurchaseTaskLineUpdatePostReq.dart
import 'package:tiq/models/dtos/PurchaseTaskLineUpdateDto.dart';

class PurchaseTaskLineUpdatePostReq {
  String purchaseTaskId;
  String productBarcode;
  String currentPalletCode;
  PurchaseTaskLineUpdateDto purchaseTaskLineUpdate;

  PurchaseTaskLineUpdatePostReq(this.purchaseTaskId, this.productBarcode, this.currentPalletCode, this.purchaseTaskLineUpdate);

  toJson() {
    return {
      'purchaseTaskId': purchaseTaskId,
      'productBarcode': productBarcode,
      'currentPalletCode': currentPalletCode,
      'purchaseTaskLineUpdate': purchaseTaskLineUpdate.toJson(),
    };
  }
}